README.md

```diff
@@ -28,16 +35,57 @@ This section describes the terms used for all documentation in this repository.
   create an environment to host the container. Neither the guest OS nor the
   guest kernel need to be the same as the host operating system.
 
-  See [the image builder documentation](image-builder/README.md).
+  See the [image creation](#image-creation) section.
 
 - initrd (or "initramfs")
 
-  A compressed cpio archive loaded into memory and used as part of the Linux
-  startup process. During startup, the kernel unpacks it into a special
-  instance of a tmpfs that becomes the initial root file system.
+  A compressed `cpio(1)` archive, created from a rootfs which is loaded into memory and used as part of the Linux startup process. During startup, the kernel unpacks it into a special instance of a `tmpfs` that becomes the initial root filesystem.
 
-  See [the initrd builder documentation](initrd-builder/README.md).
+  See the [initrd creation](#initrd-creation) section.
 
 - "Base OS"
 
-  A particular version of a Linux distribution used to create a Guest OS from.
+  A particular version of a Linux distribution used to create a rootfs from.
+
+## Usage
+
+The top-level `Makefile` contains an example of how to use the available components.
+
+By default, components will run on the host system. However, some components
+offer the ability to run from within Docker (for ease of setup) by setting the
+`USE_DOCKER=true` variable.
+
+For more detailed information, consult the documentation for a particular component.
+
+### Rootfs creation
+
+To build a basic rootfs using the default distribution:
+
+```
+$ sudo -E PATH=$PATH make USE_DOCKER=true rootfs
+```
+
+For further details, see
+[the rootfs builder documentation](rootfs-builder/README.md).
+
+### Image creation
+
+To create an image from the already-created rootfs:
+
+```
+$ sudo -E PATH=$PATH make USE_DOCKER=true image-only
+```
+
+For further details, see
+[the image builder documentation](image-builder/README.md).
+
+### Initrd creation
+
+To create an initrd from the already-created rootfs:
+
+```
+$ sudo -E PATH=$PATH make initrd-only
```

Comment:
I think one should also specify `AGENT_INIT=true` when building both the initrd image and the rootfs for it. I'm not sure about how systemd works in initramfs. Maybe we should consolidate the options to just provide three:
1. image with systemd as init
2. image with kata-agent as init
3. initrd with kata-agent as init

Reply:
Good catch! Branch updated.